project/plugins.sbt

```diff
@@ -1,12 +1,10 @@
-resolvers += "HMRC-open-artefacts-maven" at "https://open.artefacts.tax.service.gov.uk/maven2"
-resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(
+resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
+resolvers += Resolver.url("HMRC-open-artefacts-ivy2", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(
   Resolver.ivyStylePatterns
 )
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.14.0")
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.13.0")
-
-addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.4.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-auto-build"     % "3.6.0")
+addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "2.1.0")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin"         % "2.8.15")
```

Comment:
I don't think we should need `sbt-plugin` and `sbt-distributable`

Reply:
Removed